routing/endpointregistry.go

```diff
@@ -22,12 +22,25 @@ type Metrics interface {
 	InflightRequests() int64
 	IncInflightRequest()
 	DecInflightRequest()
+
+	IncRequests(o IncRequestsOptions)
+	HealthCheckDropProbability() float64
+}
+
+type IncRequestsOptions struct {
+	FailedRoundTrip bool
 }
 
 type entry struct {
 	detected         atomic.Value // time.Time
 	lastSeen         atomic.Value // time.Time
 	inflightRequests atomic.Int64
+
+	minRequests                int64
```

Comment:
The same minRequests in each entry does not seem right

Reply:
https://github.com/zalando/skipper/pull/2888#discussion_r1499033713
It was the only place where it was used. As I will remove that code, this field is not needed anymore.